src/pages/home-pages/home-pages.pages.jsx

```diff
@@ -133,3 +133,20 @@ export const HomePage4 = () => {
     </div>
   );
 };
+
+export const HomePage5 = () => {
+  return (
+    <div id="api-authentication" className='main-container'>
+      <h2> Api Authentication </h2>
+
+      <p>Authentication involves identifying/verifying who is requesting access to your api whilst authorization is determining the degree of access that is available for the user, i.e. verifying what specific files, applications, data the user can access.</p>
```

Comment:
This is not all the information that was in the doc agreed please add all the information in the relevant sub section